data/prefs.glade

```diff
@@ -151,13 +151,13 @@
                                         <property name="visible">True</property>
                                         <property name="can_focus">True</property>
                                         <property name="has_tooltip">True</property>
-                                        <property name="tooltip" translatable="yes">Path to a bash script that would be automatically executed when guake starts, unless you specify --no-startup-script.
+                                        <property name="tooltip" translatable="yes">Path to a bash script that would be automatically executed when Guake starts, unless you specify --no-startup-script.
 
-This would typically use the configuration by command line feature of guake:
+This would typically use the configuration by command line feature of Guake:
 
 #!/bin/bash
 
-sleep 5  # it is advised to wait a bit guake has been successfully started
+sleep 5  # it is advised to wait a bit after Guake has been successfully started
```

Comment:
Please replace it with "before" in a subsequent commit - I'm new to github and I don't know how to change my pull request.